### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by plotkiny**
Added the following features to the codebase:

1. Imputation of data can now be done by: mean, median, KNN
2. Added cosine similarity metric to KNN.
3. Standardization of data using both Z-score normalization and Max-Min scaling.

Updates to running the codebase:

1. Converted get_neighbors method in KNN class to a static method.

### CODE REVIEW

**Comment by sidnarayanan** [comment]
Sorry @plotkiny - I somehow missed this PR. Thanks for the contribution! Do you mind just getting rid of those crunchbangs in the library files? I'll make inline comments.

File: braf/braf.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```
Comment (sidnarayanan) [inline braf/braf.py]: e.g. here - could you drop these in any non-executable file?

Comment (plotkiny) [reply braf/braf.py]: glad to contribute. done.

Comment (plotkiny) [reply braf/braf.py]: added slight update to imputed_vals dict for knn imputation algo

**Comment by plotkiny** [comment]
Hi Sidnarayanan,

Updates made to the develop branch. Let me know what you think.

Cheers,
Yuri

**Comment by sidnarayanan** [comment]
Thanks @plotkiny ! Merging.

**Pull request merged**